### ISSUE TITLE
Dependabot configured, but shows 'Dependabot version updates aren't configured yet'

### ISSUE THREAD

**Issue by ianrathbone** [dependabot/dependabot-core#6600]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Package ecosystem

npm,yarn

### Manifest location and content before the Dependabot update

https://github.com/4Com/vue-azure-app-configuration/blob/main/package.json

### dependabot.yml content

https://github.com/4Com/vue-azure-app-configuration/blob/main/.github/dependabot.yml

### What you expected to see, versus what you actually saw

Depanabot showing as configured in GitHub, but it is not

**Comment by ianrathbone** [comment]
Related to #5112 - raised this new issue as requested

**Comment by jeffwidman** [comment]
Thanks for creating a new issue. 👍 

The problem is with a misconfiguration in your `dependabot.yml` file: https://github.com/4Com/vue-azure-app-configuration/runs/11099417621

Ideally that CI check would auto-run on the PR branch and not just `main`, but that's blocked by:
* https://github.com/dependabot/dependabot-core/issues/4605

**Comment by ianrathbone** [comment]
Apologies, completely missed that and thank you for your help!

**Comment by jeffwidman** [comment]
No worries. I look forward to when that CI check can run on PRs and flags this front and center for users.
